src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,313 @@
+---
+part_number: II
+chapter_number: 13
+title: Capabilities
+description: Capabilities chapter of the 2020 Web Almanac covering brand-new, powerful web platform APIs.
+authors: [christianliebel]
+reviewers: [logicalphase,tomayac]
+analysts: [tomayac]
+translators: []
+discuss: 1981
```

Comment:
We have the 2020 discussion ids now and [your own is 2049](https://discuss.httparchive.org/t/chapter-13-capabilities/2049):

```suggestion
discuss: 2049
```